craft_providers/lxd/remotes.py

```diff
@@ -144,6 +144,12 @@ def add_remote(self, lxc: LXC) -> None:
         remote_address=BUILDD_RELEASES_REMOTE_ADDRESS,
         remote_protocol=ProtocolType.SIMPLESTREAMS,
     ),
+    ubuntu.BuilddBaseAlias.NOBLE: RemoteImage(
+        image_name="core24",
```

Comment:
We're unable to use the `buildd` daily image of 24.04 (which has the `core24` alias) because of this outstanding bug: https://bugs.launchpad.net/cloud-images/+bug/2007419

As a workaround, we've been using the regular daily builds (which don't have the `core` aliases).

Until that launchpad bug gets resolved, we have to use an alias available for the daily images:

```suggestion
        image_name="noble",
```

Reply:
We're still waiting CPC to fix the bug mentioned above.  I will merge this PR as soon as that bug is resolved.

Reply:
Related: we're also waiting on that bug to land https://github.com/canonical/craft-providers/pull/482

Reply:
the bug in question was fixed in noble, not yet SRUed. let me check where that is at.